packages/common/src/content/_internal/ContentBusinessRules.ts

```diff
@@ -17,6 +17,7 @@ export const ContentPermissions = [
   "hub:content:delete",
   "hub:content:edit",
   "hub:content:view",
+  "hub:content:canEditAccess",
```

Comment:
not a big thing, but I was scratching my head for a second to understand what this was actually doing... maybe `:canChangeAccess`?